src/test/java/com/uw/util/UserNameGeneratorTest.java

```diff
@@ -1,11 +1,18 @@
 package com.uw.util;
 
-import com.uw.dao.TraineeDao;
 import com.uw.dao.TraineeDaoImpl;
-import com.uw.dao.TrainerDao;
 import com.uw.dao.TrainerDaoImpl;
 import com.uw.model.Trainee;
 import com.uw.model.Trainer;
+import com.uw.model.User;
+import com.uw.service.TraineeService;
+import com.uw.service.TraineeServiceImpl;
+import com.uw.service.TrainerService;
+import com.uw.service.TrainerServiceImpl;
+import org.hibernate.SessionFactory;
```

Comment:
Quick question for this ant other tests. Are you tests writing in a temporal db? or are writing on the real one?. If the second then I would recommend to use Junit Mockito for DB simulation. By the way not part of this activity but as takeaway for upcoming tasks